examples/chip-tool/commands/clusters/ModelCommand.h

```diff
@@ -69,6 +69,7 @@ class ModelCommand : public CHIPCommand
 
     virtual void ClearICDEntry(const chip::ScopedNodeId & nodeId);
 
+    virtual void AddICDEntry(chip::app::ICDClientInfo & clientinfo, chip::ByteSpan key);
```

Comment:
Why is this virtual?  It doesn't need to be virtual.

It looks like ClearICDEntry is also incorrectly virtual....

Reply:
removed, thanks